plugins/org.obeonetwork.m2doc.doc/doc/user/M2Doc User Guide.html

```diff
@@ -173,6 +173,17 @@ <h4 id="Siriustableproviders">Sirius table providers</h4>
 			<b>Warning</b>: do not forget the single quotes ' around the string in an AQL Expression: 
 			<code>title:"'Table name'"</code>. The description ID does not need such quotes because it is of type String.
 		</p>
+		<h4 id="UpdatestrategyforSiriusRepresentation">Update strategy for Sirius Representation</h4>
+		<p>In both case Sirius diagram and Sirius table providers, the M2Doc generation can occur with a closed Sirius representation view. So the semantic and graphical models could be unsynchronized. A parameter in genConf model define the strategy to follow during M2Doc generation.</p>
+		<p>
+			<img border="0" src="images/image019.png"/>
+		</p>
+		<p>The boolean parameter &#171;Refresh Diagram&#187; can be used to define following update strategy : 
+			<br/>- if it is set to false, the M2Doc generation take the state of graphical model without update it
+			<br/>- if it is set to true, the M2Doc generation take the updated with semantic model and arranged model like if we open diagram or table in Sirius representation view.
+			<br/>By default, the parameter is set to false.
+		</p>
+		<p>Becarefull, enabling update representation could be expensive in term of time machine. </p>
```

Comment:
Be careful, enabling the automatic update of representations can be time-consuming and could have a high impact on the generation time, especially on large models.